tweets/glitch-update-15-03-19.tweet

```diff
@@ -0,0 +1,6 @@
+Thanks to @melissamcewen for providing a shiny new README that shows off how easily you can get started writing GitHub Apps with @Glitch!!
```

Comment:
Maybe "... writing Probot Apps with @Glitch!!" or "... writing GitHub Apps with @Glitch and Probot!!"?

I’d also just do one "!", someone told me once that multiple exclamation marks reads like SCREAMING :)

Reply:
oh "with me" is even nicer 👉🤖👈